test/distributed/org/apache/cassandra/distributed/test/accord/AccordDropKeyspaceTest.java

```diff
@@ -35,8 +34,7 @@ public void dropKeyspace() throws IOException
         int steps = 5;
         try (Cluster cluster = Cluster.build(3)
                                       .withoutVNodes()
-                                      .withConfig(c -> c.with(Feature.values())
```

Comment:
please revert, `Feature.values()` is 100% intentional

Reply:
When I merged I set it explicitly to `GOSSIP, NETWORK, NATIVE_PROTOCOL` but we should probably go back to `values` if the tests can run fine now that we know that `BLANK_GOSSIP` is ignored.